Modules/WunderBar/SubModules/DataBar.lua

```diff
@@ -13,7 +13,7 @@ local C_Reputation_GetFactionParagonInfo = C_Reputation.GetFactionParagonInfo
 local C_Reputation_IsFactionParagon = C_Reputation.IsFactionParagon
 local CreateFrame = CreateFrame
 local format = string.format
-local GetFriendshipReputation = GetFriendshipReputation
+local GetFriendshipReputation = C_GossipInfo.GetFriendshipReputation
```

Comment:
I'm betting that `C_GossipInfo` needs to be added to the `.luacheckrc` file as a `global`. If you look for `C_QuestLog` in that file, it'll probably be within that same block.

Reply:
Sweet!